docs/devGuide/development/migratingNodeJs.md

```diff
@@ -0,0 +1,97 @@
+<box type="info" seamless>
+If a different version of npm is needed this can be overridden. This will be explained further below.
+</box>
+
+#### Setup 
+
+Install <tooltip content="Node Version Manager">[nvm](https://github.com/nvm-sh/nvm)</tooltip> to help switch between different node versions.
+Switch to use the node version that you are migrating to.
+
+### Migration steps 
+
+1. Refactor any deprecated syntax
+   - Go to the [Node.js changelog](https://nodejs.org/en/blog/release) of the new version
+   - Go through list of deprecated syntax and check if it is being used in Markbind
+   - Replace any deprecated syntax
+
+2. Check that all user-facing functionalities are working 
+   - A quick way to do this is to go to the [Reader Facing Features in the User Guide]({{baseUrl}}/userGuide/readerFacingFeatures.html)
+      and check that these features are still working as expected.
+
+3. Check that there are no issues with development setup 
+   - Set up the development environment by running through the steps in [Setting Up]({{baseUrl}}/devdocs/devGuide/development/settingUp.html) to ensure there are no problems
+
+4. Update GitHubActions 
+   - Go to [Markbind/markbind-action](https://github.com/MarkBind/markbind-action) and update the node version numbers
+     - See [Update node version from 14 to 16 PR](https://github.com/MarkBind/markbind-action/pull/8/files) for an example
+   - Test there are no issues with workflows
+     - Testing instructions located here: [markbind-action]({{baseUrl}}/devdocs/devGuide/githubActions/markbindAction.html) and [markbind-reusable-workflows]({{baseUrl}}/devdocs/devGuide/githubActions/markbindReusableWorkflows.html)
+
+<box type="info" seamless header="If a different npm version is needed">
+
+Install correct version of npm in `action.yml` and `fork-build.yml`. Refer to [Update node version from 14 to 16 PR](https://github.com/MarkBind/markbind-action/pull/8/files) to see where npm install should be run.
+</box>
+
+5. Check deployment to Netlify/other platforms
+   - Deployment to Netlify 
+     - Follow steps in [Deploying to Netlify]({{baseUrl}}/userGuide/deployingTheSite.html#deploying-to-netlify) but change the `NODE_VERSION` value accordingly. Check there are no issues with deployment and deployed site is as expected.
+     - Markbind has two repos [init-minimal-netlify](https://github.com/MarkBind/init-minimal-netlify) and [init-typical-netlify](https://github.com/MarkBind/init-typical-netlify) which allows deployment to Netlify by using a config file. 
+     Update the config file `netlify.toml` with the correct node version and check that deployment using button in `README` works as expected.
+<box type="info" seamless header="If a different npm version is needed"> 
+
+To specify the npm version add an environment variable `NPM_VERSION` with the correct version number. 
+</box>
+
+   - Deployment to Github pages
```

Comment:
The indentation here is a bit off - I think it's rendering as one level less than intended